### PR TITLE
[RELEASE-BLOCKING] - Fix positioning of agent controls

### PR DESCRIPTION
**Pull request by znicholasbrown**
PR Checklist:

- [ ] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
Fixes an issue with the current master branch where agent tab controls are hidden due to a recent nav changes.

### CODE REVIEW

**Comment by whimsicallyson** [comment]
Screen Shot 2020-11-30 at 10 41 00 AM
I'm seeing overlap on smaller screen sizes

File: src/components/Agents/Agents.vue [hunk 1]
```diff
@@ -465,16 +465,23 @@ export default {
...
-  right: 8px;
-  z-index: 1;
+  left: 50%;
+  position: fixed;
+  transform: translate(570px);
+  z-index: 4;
 
   &.md-and-down {
-    top: -124px;
+    top: 146px;
```
Comment (whimsicallyson) [inline src/components/Agents/Agents.vue:474]: changing this to ~120px should fix the overlap

**Review by whimsicallyson — CHANGES_REQUESTED** [review]

File: src/components/Agents/Agents.vue (hunk 1 above)
Comment (znicholasbrown) [reply src/components/Agents/Agents.vue:474]: Ok should be fixed now, had to add another media query ti capture the sm-md breakpoint.

**Comment by whimsicallyson** [comment]
this works!

**Review by whimsicallyson — APPROVED** [review]

**Pull request merged**